test/tune_example.py

```diff
@@ -55,3 +65,4 @@ def train_lgbm(config: dict) -> dict:
     time_budget_s=3,
     num_samples=-1,
 )
+print(analysis.best_result)
```

Comment:
print(analysis.best_config) ?

analysis.best_results throws _`AttributeError: 'ExperimentAnalysis' object has no attribute 'best_result'`_ for me

Reply:
Thanks. I added the `best_result` property.